src/components/10-atoms/checkbox/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.1.0
+
+A new attribute `variant` was added. This makes it possible to switch between a checkbox with a square or a tick.
```

Comment:
`with a square or a tick` is a bit misleading. I would write something like: 

`This makes it possible to switch between a checkbox with a checkmark or an inner square as select-indicator.`

Minor and not sure if my approach is good.

Reply:
I like it 👍